cli/src/main/java/com/devonfw/tools/ide/npm/NpmJsonDist.java

```diff
@@ -7,6 +7,6 @@
  *
  * @see NpmJsonObject#versions()
  */
-public record NpmJsonDist(@JsonProperty("tarball") String tarball) {
+public record NpmJsonDist(@JsonProperty("tarball") String tarball, @JsonProperty("shasum") String sha1) {
```

Comment:
Isn't `sha1` outdated?

Reply:
For final freedom I removed the property and its related changes again.
We currently do not need it.
When we download from NPM registry in the future, we can add it then.